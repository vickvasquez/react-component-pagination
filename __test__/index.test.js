import React from 'react'
import renderer from 'react-test-renderer'
import Pagination from '../build'

it('It renderer without errors', () => {
    const component = renderer.create(<Pagination
        page={1}
        pages={10}
        onClick={page => console.log(page)}
    />)

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
})
