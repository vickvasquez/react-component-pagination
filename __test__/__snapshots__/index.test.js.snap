// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renderer without errors 1`] = `
<ul
  className="index__pagination___sk_9O"
>
  <button
    className="index__itemPagination___S-Ybn"
    disabled={true}
    onClick={[Function]}
  >
    &lt;
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    disabled={true}
    onClick={[Function]}
  >
    &lt;&lt;
  </button>
  <button
    className="index__pageCurrent___-4MRX"
    onClick={[Function]}
  >
    1
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    onClick={[Function]}
  >
    2
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    onClick={[Function]}
  >
    3
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    onClick={[Function]}
  >
    4
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    onClick={[Function]}
  >
    5
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    onClick={[Function]}
  >
    6
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    onClick={[Function]}
  >
    7
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    onClick={[Function]}
  >
    8
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    onClick={[Function]}
  >
    9
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    onClick={[Function]}
  >
    10
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    disabled={false}
    onClick={[Function]}
  >
    &gt;&gt;
  </button>
  <button
    className="index__itemPagination___S-Ybn"
    disabled={false}
    onClick={[Function]}
  >
    &gt;
  </button>
</ul>
`;
